crates/algokit_transact/src/test_utils/mod.rs

```diff
@@ -36,42 +37,24 @@ impl TransactionHeaderMother {
                     .try_into()
                     .unwrap(),
             )
+            .fee(1000)
             .to_owned()
     }
 
-    pub fn simple_testnet_payment() -> TransactionHeaderBuilder {
+    pub fn simple_testnet() -> TransactionHeaderBuilder {
         Self::testnet()
-            .transaction_type(TransactionType::Payment)
```

Comment:
I'm a bit confused on why/how we're removing the transaction type here. Is it payment by default?

Reply:
Ahhh ok that makes sense. I was missing the fact that this was just for the header